client-app/src/examples/news/AppComponent.ts

```diff
@@ -1,18 +1,24 @@
-import {a} from '@xh/hoist/cmp/layout';
+import {a, p} from '@xh/hoist/cmp/layout';
 import {relativeTimestamp} from '@xh/hoist/cmp/relativetimestamp';
-import {hoistCmp, uses} from '@xh/hoist/core';
+import {XH, hoistCmp, uses} from '@xh/hoist/core';
 import {appBar, appBarSeparator} from '@xh/hoist/desktop/cmp/appbar';
 import {panel} from '@xh/hoist/desktop/cmp/panel';
 import {Icon} from '@xh/hoist/icon';
 import {AppModel} from './AppModel';
 import {newsPanel} from './NewsPanel';
 import '../../core/Toolbox.scss';
+import {errorMessage} from '@xh/hoist/desktop/cmp/error';
 
 export const AppComponent = hoistCmp({
     displayName: 'App',
     model: uses(AppModel),
 
     render({model}) {
+        let newsErrorThrown = model.newsPanelModel.newsErrorThrown;
+        if (newsErrorThrown && !newsErrorThrown['isRoutine']) {
+            XH.handleException(newsErrorThrown, {showAlert: false, showAsError: false});
```

Comment:
don't want to 'handle' exception in render method -- not looking to do any state changing methods during render cycle.  Move this handling to the model.